action/entry.php

```diff
@@ -217,17 +217,17 @@ protected function validate(AbstractBaseType $type, $label, $data) {
                     }
                 }
             }
-        } else {
-            if(!blank($data)) {
-                try {
-                    $type->validate($data);
-                } catch(ValidationException $e) {
-                    msg($prefix . $e->getMessage(), -1);
-                    $ok = false;
-                }
-            }
+            return $ok;
         }
 
+        if(!blank($data)) {
+            try {
+                $type->validate($data);
+            } catch(ValidationException $e) {
+                msg($prefix . $e->getMessage(), -1);
+                $ok = false;
+            }
+        }
```

Comment:
Why did you move this outside the else? Wouldn't multivalue types be validated twice now? A non-empty array is !blank.